test/plugin/scenarios/hutool-http-5.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/hutoolhttp5/controller/CaseController.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.testcase.hutoolhttp5.controller;
+
+import cn.hutool.http.HttpUtil;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/case")
+@Log4j2
+public class CaseController {
+
+    private static final String SUCCESS = "Success";
+
+    @RequestMapping("/hutool-http-5.x-scenario")
+    @ResponseBody
+    public String testcase() {
+        // your codes
+        String result = HttpUtil.get("http://localhost:8080/hutool-http-5.x-scenario/case/hutool");
+        return SUCCESS + ":" + result;
+    }
+
```

Comment:
@looly It was a long time since we last talked. 

I am glad to see SkyWalking has a new plugin contribution for your project. This part of PR is a demo application for using hutool HTTP. Could you take a look? Is this a good practice for HttpUtil?

You could find the expected traces/spans in `expectedData.yaml`